### DISCUSSION TITLE
Handling responsiveness

### DISCUSSION THREAD

**Discussion by JesperKawuma** [Open-Science-Community-Saudi-Arabia/OSCSA_Website#171] [Ideas]
Hello @BatoolMM, I would suggest that we include more responsiveness by adding Tablet view to encounter for all devices.

**Comment by BatoolMM** [comment]
@JesperKawuma Feel free to open an issue and PR to address this!

**Comment by JesperKawuma** [comment]
ok @BatoolMM, let me work on that.
